### ISSUE TITLE
java.lang.ClassNotFoundException: Didn't find class "android.permission.POST_NOTIFICATIONS"

### ISSUE THREAD

**Issue by lsantaniello** [dpa99c/cordova-plugin-firebasex#818]
In my Ionic 7 App with cordova engine, 
if I add into config.xml this code

`
<uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
`

App crash at startup with error:

`java.lang.ClassNotFoundException: Didn't find class "android.permission.POST_NOTIFICATIONS"`

**Comment by kaeverens** [comment]
I solved something similar in Cordova today. It looks like the build system caches references to classes, and sometimes it might not clean up correctly as you add/remove plugins, so at some point you might add a plugin and the build system doesn't add the class because it believes it's already added. solution: nuke it from orbit. clean out the build artifacts and the platform, remove all plugins, re-add them, re-add the platform, and then recompile.

basically "turn it off and on again"

**Comment by dpa99c** [comment]
Closing as not specifically caused by this plugin
